bin/parse_pom_xml

```diff
@@ -22,17 +22,28 @@ if parsed_file.errors.size.positive?
   exit(1)
 end
 
-dependencies = []
+properties = {}
+# Parse contents within each <properties> tag
+nodes = parsed_file.css('properties')
+nodes.each do |node|
+  childrens = node.children
+  childrens.each do |children|
```

Comment:
This is 2 levels of nesting.  Could there be more than 2?

Reply:
Nope, so should be good here.